### DISCUSSION TITLE
SVG capture

### DISCUSSION THREAD

**Discussion by thijsmie** [ewels/rich-click#74] [Q&A] [answered]
Hi its me again!

I was thinking, can we build something to capture the SVG output from the Console, as Rich currently supports, so I can include images of the help pages of the various commands in my Sphinx docs? I currently monkeypatch the Console object to enable the capturing, then run the help command and save the image, which works but is not pretty.

**Comment by ewels** [comment, marked as answer]
So, funny you should say that.. I have a new package up my sleeve for doing exactly this. I wanted to get in the core features before I made it public, but it might be time to come out of stealth mode soon 🤫 👀 

I'll add you to the private repo for some beta testing if you like 👍🏻

**Reply by thijsmie** [reply]
Great, thanks! I'll give it a try :smile:!

**Reply by ewels** [reply]
Bit late sorry, but here it is! 🎉 https://github.com/ewels/rich-codex

**Reply by thijsmie** [reply]
Hi @ewels! Yeah I've been using it since release without issues 😄

**Reply by ewels** [reply]
Great! Yes was mostly posting for future Googlers really :) If you fancy marking it as answered that would be fab 👍🏻
